paddle/common/flags.cc

```diff
@@ -1368,6 +1368,18 @@ PHI_DEFINE_EXPORTED_bool(
     false,
     "EinsumOp backward will be speedup at the expense of more gpu memory.");
 
+/**
+ * Performance related FLAG
+ * Name: enable_auto_layout_pass
+ * Since Version: 2.6.0
```

Comment:
```suggestion
 * Since Version: 3.0.0
```
准确的讲，这里应该写下一个要发布的正式版本